app/config/settings.py

```diff
@@ -21,7 +21,7 @@ class Settings(BaseSettings):
     ACCESS_TOKEN_EXPIRE_MINUTES: int = os.environ.get("ACCESS_TOKEN_EXPIRE_MINUTES")
     REFRESH_TOKEN_EXPIRE_HOURS: int = os.environ.get("REFRESH_TOKEN_EXPIRE_MINUTES")
     REFRESH_TOKEN_EXPIRE_HOURS_LONG: int = os.environ.get("REFRESH_TOKEN")
-
+    # PASSWORD_RESET_TOKEN_EXPIRE_MINUTES: int = os.environ.get("PASSWORD_RESET_TOKEN_EXPIRE_MINUTES")
```

Comment:
why did you comment out

Reply:
Cuz it's not needed yet and it was the reason the build was failing earlier.

Reply:
add it to the pytest.ini

Reply:
add it to the pytest.ini

Reply:
add it to the `pytest.ini`